app/views/paymentOptions.html

```diff
@@ -0,0 +1,18 @@
+<div class="sub-page-title-block">
```

Comment:
Yes, right now the only way to access paymentOptions.html is from the SMS link.  The CitationInfo page has a "Pay My Ticket" button which will link to an actual court approved payment website.